paymentsheet/src/test/java/com/stripe/android/paymentsheet/verticalmode/ManageScreenUITest.kt

```diff
@@ -141,22 +149,26 @@ class ManageScreenUITest {
         }
 
     @Test
-    fun clickingPaymentMethod_inEditMode_doesNothing() =
+    fun clickingPaymentMethod_inEditMode_useNewUpdateCardScreen_updatesCard() =
```

Comment:
this test was a duplicate of the one above it, so I just modified it to test the behavior with the new feature flag enabled